hnn_core/basket.py

```diff
@@ -36,6 +36,24 @@ def __set_props(self, cell_name, pos):
             'name': cell_name,
         }
 
+    # For all synapses, section location 'secloc' is being explicitly supplied
+    # for clarity, even though they are (right now) always 0.5.
+    # Might change in future
+    # creates a RECEIVING inhibitory synapse at secloc
+    def syn_gabaa_create(self, secloc):
```

Comment:
moved these here so it is clear that these methods are used only in the basket cell object

Reply:
Good call. This will definitely make things clearer when someone decides to implement a different version of the network model where a cell type is different.